tests/annotations/tests.py

```diff
@@ -182,6 +182,12 @@ def test_filter_wrong_annotation(self):
                 sum_rating=Sum('rating')
             ).filter(sum_rating=F('nope')))
 
+    def test_combined_annotation_reflection(self):
+        # see #24508
```

Comment:
It's much nicer to include a doc string with an explanation of the issue (if it can be done simply enough) rather than requiring the reader to lookup the ticket (also include the ticket number if there is further background/explanation in the ticket).